graph/src/data/schema.rs

```diff
@@ -665,11 +680,32 @@ impl Schema {
                 )
             })?;
         if field.is_derived() {
-            let derived_from = field.find_directive("derivedFrom").unwrap();
+            let derived_from = field.find_directive("derivedfrom").unwrap();
```

Comment:
It should really be spelled `derivedFrom`

Reply:
Thanks, it was a typo while adding the `&Field` return.